### PR TITLE
PID File generation for Kafka Monitoring

### PR DESCRIPTION
**Pull request by piyushchordia**

### CODE REVIEW

**Comment by piyushchordia** [comment]
@bbaugher Please have a check

File: templates/default/service.erb [hunk 1]
```diff
@@ -32,6 +32,11 @@ STOP_SERVER_CMD="$BIN_DIR/$SCRIPT-stop 2> /var/log/confluent/$NAME-service.out"
 export <%= k %>="<%= v %>"
 <% end -%>
 
+##For Kafka Monitoring
+KAFKA_PIDFILE=/var/run/$NAME.pid
+mkdir -p $(dirname $KAFKA_PIDFILE)
+chown $USER $(dirname $KAFKA_PIDFILE) || true
```
Comment (bbaugher) [inline templates/default/service.erb]: I don't think we want the confluent user owning `/var/run`. I believe its more typical to create a directory in `/var/run` which is owned by the service user. That creation logic should be moved into the `_install` recipe

File: templates/default/service.erb [hunk 2]
```diff
@@ -48,6 +53,11 @@ start() {
   else
     echo "$NAME is already running"
   fi
+#Generating PID File
```
Comment (bbaugher) [inline templates/default/service.erb]: Indent to match function

**Comment by piyushchordia** [comment]
@bbaugher Have made suggested changes, please review

**Comment by bbaugher** [comment]
Looks good, thanks

**Pull request merged**